File: applicatioan-1/ogrencibilgisistemi-python/ogrenci.py
class Ogrenci:
    def __init__(self, ad, soyad, numara, bolum, fakulte, giris_yili, sinif):
        self.ad = ad
        self.soyad = soyad
        self.numara = numara
        self.bolum = bolum
        self.fakulte = fakulte
        self.giris_yili = giris_yili
        self.sinif = sinif

    def get_ad_soyad(self):
        return f"{self.ad} {self.soyad}"

File: applicatioan-1/ogrencibilgisistemi-c#/OgrenciBilgiSistemi/Ogrenci.cs
public class Ogrenci
{
    public string Ad { get; private set; }
    public string Soyad { get; private set; }
    public int Numara { get; private set; }
    public string Bolum { get; private set; }
    public string Fakulte { get; private set; }
    public int GirisYili { get; private set; }
    public int Sinif { get; private set; }

    public Ogrenci(string ad, string soyad, int numara, string bolum, string fakulte, int girisYili, int sinif)
    {
        Ad = ad;
        Soyad = soyad;
        Numara = numara;
        Bolum = bolum;
        Fakulte = fakulte;
        GirisYili = girisYili;
        Sinif = sinif;
    }

    public string GetAdSoyad()
    {
        return $"{Ad} {Soyad}";
    }
}
